raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -345,6 +331,18 @@ library NettingChannelLibrary {
         selfdestruct(0x00000000000000000000);
     }
 
+    function index_or_throw(Data storage self, address participant_address)
+        view
```

Comment:
Since introductions of `view` and `pure` it seems that `view` is essentially [an alias](http://solidity.readthedocs.io/en/develop/contracts.html?highlight=pure#view-functions) for `constant`.

Let's choose either `constant` or `view` and stick to it. I can go with either, but prefer `constant` due to it being more familiar from other languages.